### PR TITLE
Fix typo all

### PR DESCRIPTION
**Pull request by whs-dot-hk**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind cleanup

**What this PR does / why we need it**:

Fix typo all

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_history.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func NewCmdRolloutHistory(f cmdutil.Factory, streams genericclioptions.IOStreams
 	return cmd
 }
 
-// Complete completes al the required options
+// Complete completes all the required options
```
Comment (johscheuer) [inline staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_history.go]: The same typo is here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_undo.go#L104 please fix this too.

**Review by johscheuer — COMMENTED** [review]
/ok-to-test

File: staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_history.go (hunk 1 above)
Comment (whs-dot-hk) [reply staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_history.go]: Aye

**Pull request closed without merging**